prov/efa/configure.m4

```diff
@@ -113,6 +114,46 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	AC_SUBST(efa_CPPFLAGS)
 	AC_SUBST(efa_LDFLAGS)
 	AC_SUBST(efa_LIBS)
+
+	AC_ARG_ENABLE([efa-unit-test],
+		[AS_HELP_STRING([--enable-efa-unit-test=CMOCKA_INSTALL_DIR],
+				[Provide a path to the CMocka installation directory
+				 in order to enable EFA Unit Tests.])])
+
+	cmocka_dir=""
+	if [test x"$enable_efa_unit_test" = x"yes"]; then
+    	cmocka_dir=""
+	else
+   		cmocka_dir="$enable_efa_unit_test"
+	fi
+
+	if [ test x"$enable_efa_unit_test" != x"" ]; then
+		FI_CHECK_PACKAGE(cmocka,
+						 [cmocka.h],
+						 [cmocka],
+						 [_expect_any],
+						 [],
+						 [$cmocka_dir],
+						 [$cmocka_dir/lib64],
+						 [
+							AC_DEFINE([EFA_UNIT_TEST], [1], [EFA unit testing])
+							efatest_LIBS=" -lcmocka -libverbs -static $efatest_LIBS"
+							efa_LIBS+=" $cmocka_LDFLAGS -lcmocka -libverbs -static $efatest_LIBS"
```

Comment:
I do not think you need to change `efatest_LIBS` and `efa_LIBS`.

Reply:
got rid of efatest_LIBS, still need efa_LIBS b/c I can't seem to drag it into Makefile.include without everything breaking.  The most important flag in there is the -static.